apps/www/pages/blog/[slug].tsx

```diff
@@ -129,8 +131,12 @@ function BlogPostPage(props: any) {
 
   const NextCard = (props: any) => {
     const { post, label, className } = props
+
+    // [Joshen] I don't necessarily think this is the right fix though
+    const formattedUrl = post.url.split('/').pop()
```

Comment:
I think [our current code](https://github.com/supabase/supabase/blob/3d14567cff0f4486b347eb9fc68b04d31b5abb58/apps/www/lib/posts.tsx#L42) adds year month and day..so should we pop multiple times?

Or just change posts.tsx to stop add dates

Reply:
hang on, this has possibly changed since

Reply:
the post.url shouldn't have any date formatting in it now

Reply:
i tried removing the date appending in lib/posts.tsx but then it messes up the URLs of the blog cards

Reply:
> so should we pop multiple times?
just once will do cause it'll pop the last item in the array

Reply:
https://github.com/supabase/supabase/pull/8253